molviewspec/molviewspec/nodes.py

```diff
@@ -516,13 +520,13 @@ class VolumeIsoSurfaceParams(RepresentationParams):
 
     type: Literal["isosurface"] = "isosurface"
 
-    relative_isovalue: Optional[float] = Field(description="Relative isovalue")
-    absolute_isovalue: Optional[float] = Field(description="Absolute isovalue. Overrides `relative_isovalue`.")
-    show_wireframe: Optional[bool] = Field(description="Show mesh wireframe. Defaults to false.")
-    show_faces: Optional[bool] = Field(description="Show mesh faces. Defaults to true.")
+    relative_isovalue: Optional[float] = Field(None, description="Relative isovalue")
+    absolute_isovalue: Optional[float] = Field(None, description="Absolute isovalue. Overrides `relative_isovalue`.")
+    show_wireframe: Optional[bool] = Field(None, description="Show mesh wireframe. Defaults to false.")
+    show_faces: Optional[bool] = Field(None, description="Show mesh faces. Defaults to true.")
 
 
-VolumeRepresentationTypeParams = {t.__fields__["type"].default: t for t in (VolumeIsoSurfaceParams,)}
+VolumeRepresentationTypeParams = {t.model_fields["type"].default: t for t in (VolumeIsoSurfaceParams,)}
```

Comment:
Do we need v1 support here?
```
VolumeRepresentationTypeParams = {
    t.model_fields["type"].default if hasattr(t, "model_fields") else t.__fields__["type"].default: t
    for t in (VolumeIsoSurfaceParams,)
}```

Reply:
Nice spot, thanks. There was one more instance of `__fields__` too.